### ISSUE TITLE
Timeout when cluster not initialized

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#871]
Originally reported of Google Code with ID 815.

```
Without initializing a cluster (i.e., gnt-cluster init) try to run some gnt commands, for example, 'gnt-cluster info'. Instead of saying that the cluster is not initialized, these commands timeout.
```
Originally added on 2014-04-28 09:06:43 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2014-04-28 09:08:20 +0000 UTC.

Added Labels: Type-Defect Priority-Medium
Added to Milestone: Release2.13

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2014-04-28 09:10:00 +0000 UTC.

Added to Milestone: Release2.12
Removed from Milestone: Release2.13

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2014-04-29 06:49:41 +0000 UTC.

Changed State: Started

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2015-07-15 13:50:30 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
I believe this has been fixed in [b7b036723a] and released in 2.12.2.
```
Originally added on 2015-07-16 12:58:41 +0000 UTC.

Changed State: Released
